WalletWasabi/Blockchain/Keys/KeyManager.cs

```diff
@@ -200,6 +207,20 @@ public static KeyManager CreateNew(Mnemonic mnemonic, string password, Network n
 		return CreateNew(seed, password, network, filePath);
 	}
 
+	public static byte[] GenerateShamirEntropy()
+	{
+		return RandomUtils.GetBytes(256 / 8);
+	}
+
+	public static KeyManager CreateNew(out Share[] shares, string password, Network network, string? filePath = null)
+	{
+		shares = Shamir.Generate(
+			DefaultShamirThreshold,
+			DefaultShamirShares,
+			GenerateShamirEntropy());
+		return CreateNew(shares.Take(DefaultShamirThreshold).ToArray(), password, network, filePath);
+	}
+
```

Comment:
This should be in the WalletGenerator .cs file

Reply:
Btw, by moving this you can undo all the changes made in all the unit tests.

Reply:
> Btw, by moving this you can undo all the changes made in all the unit tests.

https://github.com/WalletWasabi/WalletWasabi/pull/13800/commits/2371fbc0569c05145fb5c3f0e47a0cf2d10fb33c

Reply:
> Btw, by moving this you can undo all the changes made in all the unit tests

https://github.com/WalletWasabi/WalletWasabi/pull/13800/commits/c4e91486d4e0cf01d30c5f7f603e492631bc6115

https://github.com/WalletWasabi/WalletWasabi/pull/13800/commits/8463694312796b639f7264723e38ec41734ee46d

https://github.com/WalletWasabi/WalletWasabi/pull/13800/commits/968f9bff7441f1824b53a5fcbea1b156a795b458